[Bug]: Fails to configure on systems with LaTeX but missing dvisvgm and ImageMagick

### Describe the bug

```
-- I could not find the htlatex command.
-- I could not find the biber command.
CMake Error at environment/cmake/Lue.cmake:19 (message):
  dvisvgm not found, LaTeX installation not complete
Call Stack (most recent call first):
  CMakeLists.txt:82 (include)
```

now fails e.g. with EasyBuild 

### Expected behaviour

_No response_

### Steps to reproduce

_No response_

### Additional information

_No response_

--- a/environment/cmake/Lue.cmake
+++ b/environment/cmake/Lue.cmake
@@ -11,16 +11,6 @@ set(LUE_MIN_CXX_STANDARD 23)
 
 set(CMAKE_CXX_SCAN_FOR_MODULES FALSE)
 
-find_package(LATEX)
-if(LATEX_FOUND)
-    include(UseLATEX)
-    find_program(dvisvgm_EXECUTABLE dvisvgm)
-    if(NOT dvisvgm_EXECUTABLE)
-        message(FATAL_ERROR "dvisvgm not found, LaTeX installation not complete")
-    endif()
-    find_package(ImageMagick COMPONENTS convert)
-endif()
-
 # Handle configuration and 3rd party libraries and tools, and ...
 include(LueConfiguration)
 

--- a/environment/cmake/LueConfiguration.cmake
+++ b/environment/cmake/LueConfiguration.cmake
@@ -409,10 +409,10 @@ endif()
 
 
 if(LUE_BUILD_DOCUMENTATION)
-    # set(LUE_BREATHE_REQUIRED TRUE)
     set(LUE_DOXYGEN_REQUIRED TRUE)
     set(LUE_GRAPHVIZ_REQUIRED TRUE)
     set(LUE_JUPYTER_BOOK_REQUIRED TRUE)
+    set(LUE_LATEX_REQUIRED TRUE)
     set(LUE_SPHINX_REQUIRED TRUE)
     set(LUE_DOXYGEN_AWESOME_CSS_REQUIRED TRUE)
 
@@ -612,12 +612,14 @@ if(LUE_HDF5_REQUIRED)
         set(HDF5_PREFER_PARALLEL TRUE)
     endif()
 
-    # set(HDF5_FIND_DEBUG TRUE)  # Uncomment to debug HDF5's find logic
-    find_package(HDF5 REQUIRED COMPONENTS C)
+    # set(HDF5_FIND_DEBUG TRUE)  # Uncomment to debug HDF5's find logic (MODULE mode only?)
+    find_package(HDF5 MODULE REQUIRED COMPONENTS C)
 
     if(NOT HDF5_FOUND)
         message(FATAL_ERROR "HDF5 not found")
     elseif(NOT TARGET hdf5::hdf5)
+        # If this fails, consider migrating to CONFIG search mode and use HDF5's hdf5-config.cmake. It
+        # defines other targets and variables though...
         message(FATAL_ERROR "Target hdf5::hdf5 not available")
     endif()
 
@@ -657,6 +659,24 @@ if(LUE_NLOHMANN_JSON_REQUIRED)
         FIND_PACKAGE_ARGS
     )
     FetchContent_MakeAvailable(nlohmann_json)
+endif()
+
+
+if(LUE_LATEX_REQUIRED)
+    find_package(LATEX REQUIRED)
+
+    if(LATEX_FOUND)
+        include(UseLATEX)
+
+        # Needed for logo targets
+        find_program(dvisvgm_EXECUTABLE dvisvgm)
+        if(NOT dvisvgm_EXECUTABLE)
+            message(FATAL_ERROR "dvisvgm not found, LaTeX installation not complete")
+        endif()
+
+        # Needed for logo targets
+        find_package(ImageMagick REQUIRED COMPONENTS convert)
+    endif()
 endif()
 
 

--- a/environment/cmake/lue-config.cmake.in
+++ b/environment/cmake/lue-config.cmake.in
@@ -18,7 +18,7 @@ if(LUE_GDAL_REQUIRED)
 endif()
 
 if(LUE_HDF5_REQUIRED)
-    find_dependency(HDF5 COMPONENTS C)
+    find_dependency(HDF5 MODULE COMPONENTS C)
 endif()
 
 if(LUE_HPX_REQUIRED)

